Update implications_learning.md (#36)
carpentries-incubator/genai-implications-novice: episodes/implications_learning.md

---
title: "Implications for Learning"
teaching: 10 # teaching time in minutes
exercises: 10 # exercise time in minutes
---

:::::::::::::::::::::::::::::::::::::: questions 

- What is a shallow approach to learning? 
- What is a deep approach to learning?
- How does the use of GenAI for coding impact learning?
- How can GenAI be used as a tool to support critical coding rather than a shortcut?

::::::::::::::::::::::::::::::::::::::::::::::::

::::::::::::::::::::::::::::::::::::: objectives

> - Understand the difference between a shallow and a deep approach to learning

Reviewer: ```suggestion
- Understand the difference between a shallow approach and a deep approach to learning
```

> - Understand how genAI can enhance or hinder skill acquisition

Reviewer: ```suggestion
- Understand how GenAI can enhance or hinder skill acquisition
```

- Relate the implications for using GenAI to the act of learning how to code 

::::::::::::::::::::::::::::::::::::::::::::::::

## Introduction

> In this episode we will take a closer look at different approaches to learning and how they relate to using GenAI to research software development.

Reviewer 2: ```suggestion
In this episode we will take a closer look at different approaches to learning and how they relate to using GenAI in research software development.
```

::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::

## The Shallow vs Deep Approach to Learning

Consider the following questions:

1. How many legs does a spider have? 
2. How is a spider not an insect? 

:::::::::::::::::::::::::::::::::::::: discussion

### Turn to your neighbour and share your thoughts: 

1. How are these two questions different?
2. How would you go about providing answers to these questions?
3. What kind of learning takes place in each of these questions?
4. What do you think you will remember about the topics tomorrow? Next month? 

::::::::::::::::::::::::::::::::::::::::::::::::

Shallow learning, also sometimes called surface learning or rote learning, refers to learning activities that are characterized by recalling and rote memorization. The knowledge gained from shallow learning is considered passive and tends to fade away from our memory.

> Deep learning, refers to learning activities that are characterized by an effort to connect with and understand the material conceptually. The knowledge gained from deep learning is considered active. Analyzing meaning by drawing connections, elaborating on ideas, and linking them to prior knowledge leads to long-term retention in memory.

Reviewer: ```suggestion
Deep learning refers to learning activities that are characterized by an effort to connect with and understand the material conceptually. The knowledge gained from deep learning is considered active. Analyzing meaning by drawing connections, elaborating on ideas, and linking them to prior knowledge, which leads to long-term retention in memory.
```

Importantly, slow speed is an inherent characteristic of deep thinking and learning.

## Up-skill and De-skill

The notion that thinking cannot be sped up stands in stark contrast to the promise of GenAI to accelerate productivity and make things faster. Having GenAI perform a task for you is often described as cognitive offloading.

> In a [recent study](https://www.anthropic.com/research/AI-assistance-coding-skills) Anthropic sought to determine whether cognitive offloading can prevent people from growing their coding skills. In a randomized controlled trial the study found that using AI assistance led to a statistically significant decrease in mastery. 

Reviewer: ```suggestion
In a [2026 study](https://www.anthropic.com/research/AI-assistance-coding-skills), Anthropic sought to determine whether cognitive offloading can prevent people from growing their coding skills. In a randomized controlled trial the study found that using AI assistance led to a statistically significant decrease in mastery. 
```

> A deep learning approach requires to be aware of *how* you are thinking and learning. The learning process becomes a meaningful experience in and of itself. 

Reviewer: ```suggestion
A deep learning approach requires you to be aware of *how* you are thinking and learning. The learning process becomes a meaningful experience in and of itself. 
```

:::::::::::::::::::::::::::::::::::::: discussion

### Turn to your neighbour and share your thoughts: 

> 1. When you look back at a recent interaction with ChatGPT, what parts of the *thinking* did you do, and what parts did you subconsciously let the AI do? 

Reviewer 2: ```suggestion
1. When you look back at a recent interaction with an LLM, what parts of the *thinking* did you do, and what parts did you subconsciously let the AI do? 
```

2. How can you develop your ability to recognize your choices in the moment?

::::::::::::::::::::::::::::::::::::::::::::::::

> 

Reviewer 2: ```suggestion
::::::::::::::::::::::::::::::::::::: callout
In June 2026 the Norwegian government announced plans to reduce access to AI tools by learners in schools.

From August 2026 these tools would not be available for learners aged 6 to 13yrs, with those aged 14 and 16 only able to use it under direct teacher supervision.

Learners aged 17 to 19yrs will learn to "use AI appropriately" to prepare them for further education and work. 

The Norwegian Prime Minister Jonas Gahr Støre said that using AI increases the risk that young children miss important steps in their education:"The most important thing in school is that our children learn to read, write, and do mathematics,".

Source: [PC Mag UK: Norway bans AI in elementary schools](https://uk.pcmag.com/ai/165706/norway-bans-ai-in-elementary-schools)
::::::::::::::::::::::::::::::::::::::::::::::::
```

## AI Engagement Strategies

Over-reliance on AI for coding can prevent researchers from developing essential skills in research software development and data analysis. Without a solid understanding of the code it is impossible to reliably verify whether research results are correct and valid.

The study by Anthropic mentioned above also found that *how* someone used AI had a great impact on how much information they retained. Coders who used AI assistance not just to produce code but to build comprehension while doing so showed stronger mastery. 

::::::::::::::::::::::::::::::::::::: challenge 

## Challenge:

> What are some strategies to avoid cognitive offloading and instead use AI strengthen your research computing skills?

Reviewer 2: ```suggestion
What are some strategies to avoid cognitive offloading and instead use AI to strengthen your research computing skills?
```

:::::::::::::::::::::::: solution 

## Suggestions
 
Here are some strategies that could help build comprehension.

- Ask follow-up questions 
- Request explanations
- Ask for code generation along with explanations of the generated code
- Pose conceptual questions then use improved understanding to complete the task
- Ask for code review
- Use "what if" and "how/why" questions
- Introduce "friction": Instruct AI to push back, to never provide an answer outright, etc
- Take it slow!

:::::::::::::::::::::::::::::::::
::::::::::::::::::::::::::::::::::::::::::::::::

> 

Reviewer 2: ```suggestion
::::::::::::::::::::::::::::::::::::: callout

At the Association for Learning Development Conference 2025, a workshop explored how activities could be designed to enable learners to engage critically with AI. 

Read the workshop write-up:[What is lazy metacognition and what can we do about it? ](https://doi.org/10.47408/jldhe.vi37.1713 )

::::::::::::::::::::::::::::::::::::::::::::::::
```

Your overall goal should always be to focus on your learning. That means to adopt practices that help you keep learning to code and to be able to scrutinize the answers given back by GenAI. 

> GenAI output often sounds very confident, which can make us inclined to accept it as correct and factual without critically evaluating it. However, is important to treat AI outputs as suggestions rather than solutions. Also, remember that you as the researcher need to take responsibility for any AI-generated code you use. We will come back to this in the next episode.

Reviewer 2: ```suggestion
GenAI output often sounds very confident, which can make us inclined to accept it as correct and factual without critically evaluating it. However, it is important to treat AI outputs as suggestions rather than solutions. Also, remember that you as the researcher need to take responsibility for any AI-generated code you use. We will come back to this in the next episode.
```

> GenAI output often sounds very confident, which can make us inclined to accept it as correct and factual without critically evaluating it. However, is important to treat AI outputs as suggestions rather than solutions. Also, remember that you as the researcher need to take responsibility for any AI-generated code you use. We will come back to this in the next episode.

Reviewer: ```suggestion
GenAI output often sounds very confident, which can make us inclined to accept it as correct and factual without critically evaluating it, especially if we are less familiar with a topic. However, is important to treat AI outputs as suggestions rather than solutions. Also, remember that you as the researcher need to take responsibility for any AI-generated code you use. We will come back to this in the next episode.
```
